tools/integration-tests/tester/docker-compose.yml

```
@@ -3,11 +3,14 @@ version: "3.5"
 services:
   tester:
     container_name: tester
-    image: golang:1.16.2
+    image: golang:1.16
     working_dir: /tmp/goshimmer/tools/integration-tests/tester
     command: /tmp/assets/entrypoint.sh
     environment:
+      - DEBUG=${DEBUG}
       - TEST_NAME=${TEST_NAME}
+    ports:
+      - 127.0.0.1:40000:40000
```

```suggestion
      - "127.0.0.1:50000:50000"
```
1. 40000 is the default node debug port. But we are not debugging the node here so better use some other random number.
2. My IDE shouted at me for not using quotes because of some edge cases. I don't like being shouted at.

This is the only comment that caused me to press "request changes"

---

Good point.